### ISSUE TITLE
List of participating countries is misaligned

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-ios#2908]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 12 Pro Simulator 
- iOS version: 14.5
- App version: 2.3.3

## Describe the bug

The list of countries participating in the transnational exposure logging seems to be somehow misaligned in the onboarding. Compare it to the list that's shown on the "Transnational Exposure Logging" screen:
| Onboarding | Transnational Exposure Logging |
|---|---|
| <img src="https://user-images.githubusercontent.com/67682506/121709224-9ebf0e80-cad8-11eb-9a7c-826fe6edc591.png" width="300"> | <img src="https://user-images.githubusercontent.com/67682506/121709044-6dded980-cad8-11eb-8143-07eb7be65be4.png" width="300"> |

The flags are outside of the area they are supposed to be.

## Steps to reproduce the issue

1. Open a fresh installed or resetted Corona-Warn-App
2. Step through the screens until you see the screen "How You Make Exposure Logging Possible"
3. Scroll down to the list of the countries
4. See that it is somehow misaligned

## Expected behaviour

The list of the countries should be shown linke on the "Transnational Exposure Logging" screen.

## Additional context

I can't check this on a real device - please check on a real device running the latests iOS version.

----
Internal Tracking-ID: [EXPOSUREAPP-7968](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7968)

**Comment by heinezen** [comment]
@Ein-Tim 

Thanks for reporting. We have brought this to the attention of the developers by creating Jira ticket EXPOSUREAPP-7968.

----
Corona-Warn-App Open Source Team

**Comment by dsarkar** [comment]
@ndegendogo Sure! Thanks for the PR!

**Comment by Ein-Tim** [comment]
I am happy to confirm that this bug was fixed in version 2.13. 
I verified the fix on the same device I used to report this issue, the iPhone 12 Pro Simulator.

Here's a screenshot how it looks now:
<img src="https://user-images.githubusercontent.com/67682506/140173197-efdea60d-63f4-4402-b125-0c7d88d806be.png" width="300">

The list is nicely centered now and not misaligned anymore.
A thank you to @ndegendogo for fixing this issue!

Closing as fixed.
